xml/admin_ceph_upgrade.xml

```diff
@@ -361,7 +361,8 @@ mds standby for name = mds.456-another-mds
     <para>
       If you want to manually maintain the <systemitem class="daemon">chronyd</systemitem>
       configuration, follow the instructions below and ensure you disable
-      the time configuration (as describe below).
+      the time configuration. See <xref linkend="disable-time-sync"/> for more
```

Comment:
```suggestion
      <command>ntpd</command> time configuration. See <xref linkend="disable-time-sync"/> for more
```

Reply:
I am not sure [command] is the best tag. [literal] or something else might be fine too.